frontend/GoodGrocer/app/screens/Profile.js

```diff
@@ -1,79 +1,102 @@
-import React from 'react';
```

Comment:
oops did not mean to change this file but all it did was reformat it because I ran command-s so I think we're fine